test/models/rook_test.rb

```diff
@@ -1,7 +1,60 @@
 require 'test_helper'
 
 class RookTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def test_valid_vertical_move?
+    rook = Rook.new(x_position: 0, y_position: 0)
+    assert(
+      rook.valid_move?(0, 7)
+    )
+    assert(
+      rook.valid_move?(0, 2)
+    )
+  end
+
+  def test_invalid_vertical_move?
+    rook = Rook.new(x_position: 7, y_position: 7)
+    assert ! (
+      rook.valid_move?(6, 6)
+    )
+    assert ! (
+      rook.valid_move?(5, 6)
+    )
+  end
+
+  def test_valid_horizontal_move?
+    rook = Rook.new(x_position: 7, y_position: 0)
+    assert(
+      rook.valid_move?(0, 0)
+    )
+    assert(
+      rook.valid_move?(3, 0)
+    )
+  end
+
+  def test_invalid_horizontal_move?
+    rook = Rook.new(x_position: 0, y_position: 7)
+    assert ! (
+      rook.valid_move?(7, 6)
+    )
+    assert ! (
+      rook.valid_move?(1, 6)
+    )
+  end
+
+  def test_rook_off_the_board?
+    rook = Rook.new(x_position: 7, y_position: 0)
+    assert ! (
```

Comment:
there are negative `asserts` so you can check `falsey` values: http://guides.rubyonrails.org/testing.html#available-assertions